frontend/public/components/utils/list-input.tsx

```diff
@@ -81,5 +85,7 @@ type ListInputProps = {
   label: string;
   initialValues?: string[];
   onChange: ChangeCallback;
+  describedBy: string;
```

Comment:
You shouldn't need to pass this in. I'd just generated a unique value for the ID. You could use `_.unique` to generate one in the constructor:

```js
  this.helpID = _.uniqueId('list-view-help-');
```

then

```jsx
aria-describedby={helpText ? this.helpID : undefined}
```